charts/prometheus-mysql-exporter/templates/servicemonitor.yaml

```diff
@@ -12,6 +12,9 @@ metadata:
   namespace: {{ .Values.serviceMonitor.namespace }}
 {{- end }}
 spec:
+  namespaceSelector:
+    matchNames:
+    - {{ .Release.Namespace }}
```

Comment:
IMHO this should be configurable. 
Otherwise you still can only use it in the same namespace it's installed too.


See: https://github.com/prometheus-operator/prometheus-operator/blob/main/Documentation/api.md#namespaceselector

Reply:
`namespaceSelector` is used for finding metric exporter, it should be always the same as the namespace the chart installed, I don't think it needs to be customized